config/application.rb

```diff
@@ -23,10 +23,41 @@
 
 module PracticalDeveloper
   class Application < Rails::Application
-    # Initialize configuration defaults for originally generated Rails version.
-    config.load_defaults 5.1 # NOTE: [Rails 6] we should at least work towards updating this to 5.2
+    # Specify the default Rails settings version we're targetting
+    # See: https://guides.rubyonrails.org/configuring.html#results-of-config-load-defaults
+    config.load_defaults 6.0
+
+    ### FRAMEWORK DEFAULT OVERRIDES
+    # Override new framework defaults to keep existing behavior.
+    #
+    # NOTE: For booleans the new default is the opposite of what we're setting here. For other
+    # options, the new default is mentioned in a comment. Once we're ready to enable a new default
+    # setting we can remove the line here.
+
+    ## Rails 5.0
+    # There is no easy way to use per-form tokens and view caching at the same time.
+    # Therefore we disable "per_form_csrf_tokens" for the time being.
+    config.action_controller.per_form_csrf_tokens = false
+
+    ## Rails 6.0
+    # Determines whether forms are generated with a hidden tag that forces older versions of Internet
+    # Explorer to submit forms encoded in UTF-8
+    config.action_view.default_enforce_utf8 = true
+
+    # Enables the same cache key to be reused when the object being cached of type ActiveRecord::Relation
+    # changes by moving the volatile information (max updated at and count) of the relation's cache
+    # key into the cache version to support recycling cache key.
+    config.active_record.collection_cache_versioning = false
+
+    # Enables writing cookies with the purpose metadata embedded.
+    config.action_dispatch.use_cookies_with_metadata = false
+
+    ## Rails 6.1
+    # This replaces the old config.active_support.use_sha1_digests from Rails 5.2
+    config.active_support.hash_digest_class = ::Digest::MD5 # New default is ::Digest::SHA1
+    ### END FRAMEWORK DEFAULT OVERIDES
 
-    # [Rails 6] Zeitwerk is the new autoloader
+    # Zeitwerk is the new autoloader in Rails 6+
     # As we don't have `load_defaults 6.0` yet, it has to be enabled manually
```

Comment:
This comment will be outdated when this is merged

```suggestion
```

Reply:
It wasn't only the comment that was outdated, the whole block can be deleted 😃  I checked in a console, everything still works after removing the option and Rails 7 completely removes it anyway:

```ruby
Rails.application.config.autoloader
#=> :zeitwerk
```